cmd/harmony.go

```diff
@@ -82,7 +81,7 @@ func main() {
 	port := flag.String("port", "9000", "port of the node.")
 	logFolder := flag.String("log_folder", "latest", "the folder collecting the logs of this execution")
 	attackedMode := flag.Int("attacked_mode", 0, "0 means not attacked, 1 means attacked, 2 means being open to be selected as attacked")
-	dbSupported := flag.Bool("db_supported", false, "false means not db_supported, true means db_supported")
+	dbSupported := flag.Bool("db_supported", true, "false means not db_supported, true means db_supported")
```

Comment:
Two changes requested.
1) I'd ask for a parameter to let user specify where to store the db file. 
2) Also, is there a way to reload the db from disk to memory?   we should support reload the db from file to memory.